docs/markdown/Contributing.md

```diff
@@ -76,6 +76,16 @@ notes. These features should be written in standalone files in the
 `docs/markdown/snippets` directory. The release manager will combine
 them into one page when doing the release.
 
+### Skipping integration testing
+
+Integration tests are expensive and not necessary for documentation changes.
+Tests can be disabled by putting `[skip ci]` or `[ci skip]` into commit title.
```

Comment:
I think we should explicitly say that it's mandatory to put that in the title for docs changes, and we should recommend just one of the two, preferably `[skip ci]`.

Also, while reviewing PRs, we should ask people to add that to the title if it isn't already, because each commit is run in the CI twice: once when it's in PR form, and again once it's in master.